eslint.config.js

```diff
@@ -13,15 +14,25 @@ import temporarySuppressions from './eslint_temporary_suppressions.js'
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = path.dirname(__filename)
 
+const packagesPath = path.join(__dirname, 'packages')
+const packages = await fs.readdir(packagesPath)
+const packageIgnores = packages.map((name) => includeIgnoreFile(path.resolve(packagesPath, name, '.gitignore')))
```

Comment:
This is awkward. As I mentioned to @serhalp, I think we should have each package defining its own `lint` step with its own ignore rules, etc.